rnd/autogpt_server/autogpt_server/blocks/discordsendmessage.py

```diff
@@ -0,0 +1,30 @@
+import requests
```

Comment:
Probably want to call the file just `discord.py` as we can add more blocks into it in time.